### DISCUSSION TITLE
Extender board?

### DISCUSSION THREAD

**Discussion by cipeczka** [probonopd/MiniDexed#340] [General]
Ok you can close this.. 

```
I have RPI3 & PIFI Dac, what is simple way
to connect LCD display & ky040. Pifi dac dont have hat pins...

Expadner, extender ?

Please help
```

**Comment by probonopd** [comment]
If you know what are you doing, solder to the PiFi DAC pins at the top of the PCB. Of course you need to check which pins are used by the PiFi DAC itself and which ones are unused. If you don't want to do this, then you can use one of the DACs listed on the hardware page in the wiki, they only cost about 1/3rd of the PiFi DAC.

**Reply by cipeczka** [reply]
Thank You! Just ordered cheep dac from ali :)
